tests/test_batch_change.py

```diff
@@ -47,13 +48,19 @@ def test_create_batch_change(mocked_responses, vinyldns_client):
 
     drc = DeleteRecordSetChange(forward_zone.id, forward_zone.name, 'baz', 'baz.bar.com', RecordType.A, 'Complete',
                                 'id2', 'system-message', 'rchangeid2', 'rsid2')
-    bc = BatchChange('user-id', 'user-name', 'batch change test', datetime.utcnow(), [arc, drc],
-                     'bcid', 'owner-group-id')
+
+    tomorrow = datetime.utcnow() + timedelta(1)
```

Comment:
Not sure if we need to declare this separately if we are never using it afterward.